File: app/src/main/java/com/art2cat/dev/moonlightnote/controller/moonlight/TrashDetailFragment.java
package com.art2cat.dev.moonlightnote.controller.moonlight;

import com.art2cat.dev.moonlightnote.model.Moonlight;

/**
 * Created by rorschach on 11/5/16 7:08 PM.
 */

public class TrashDetailFragment extends MoonlightDetailFragment {

  public static TrashDetailFragment newInstance(Moonlight moonlight, int flag) {

    TrashDetailFragment trashDetailFragment = new TrashDetailFragment();
    trashDetailFragment.setArgs(moonlight, flag);
    return trashDetailFragment;
  }
}

File: app/src/main/java/com/art2cat/dev/moonlightnote/controller/moonlight/TrashDetailFragment.kt
package com.art2cat.dev.moonlightnote.controller.moonlight

import com.art2cat.dev.moonlightnote.model.Moonlight

/**
 * Created by Rorschach
 * on 24/05/2017 8:08 PM.
 */

class TrashDetailFragment : MoonlightDetailFragment() {
    companion object {

        fun newInstance(moonlight: Moonlight, flag: Int): TrashDetailFragment {

            val trashDetailFragment = TrashDetailFragment()
            trashDetailFragment.setArgs(moonlight, flag)
            return trashDetailFragment
        }
    }
}
